pack/tdc/tdcc.json

```diff
@@ -110,6 +110,528 @@
+        "pack_code": "tdcc",
+        "position": 55,
+        "quantity": 1,
+        "stage": 2,
+        "text": "<b>Forced</b> - When doom is placed on this agenda, if there are no non-[[Elite]] enemies in play: Reveal cards from the top of the encounter deck until an enemy is revealed and spawn it (at an empty location, if able). Shuffle each other revealed card and place them on top of the encounter deck.",
+        "type_code": "agenda"
+    },
+    {
+        "back_name": "Heart of the Apiary",
+        "back_text": "<b>If each investigator is at Lost Campsite:</b>\n<blockquote>After searching the campsite, you find yourselves surrounded by robed figures holding torches, but no weapons. A soft-faced woman in a patchwork cloak emerges from the crowd. \"Please, don't be alarmed. We have no intention of harming you.\"</blockquote>\nEach investigator loses all of their clues.\nProceed to <b>Interlude: Lost Pilgrims</b> <i>(page 21),</i>.<hr><b>If each investigator is at Acidic Coelon:</b>\n<blockquote>You hear a hiss, then feel sharp, stinging pain in your back. At you lose consciousness, look around and see apprehensive limb buried in your spine.</blockquote>/nEach investigator loses all of their clues.\nProceed to <b>Interlude: The Hive Mind</b> <i>(page 22)</i>.",
+        "code": "11556",
+        "clues": 2,
+        "double_sided": true,
+        "encounter_code": "the_apiary",
+        "encounter_position": 4,
+        "faction_code": "mythos",
+        "flavor": "The fleshy tunnels excrete a foul, waxy substance that burns the skin. Your destination lies deeper within the disgusting labyrinth.",
+        "illustrator": "Lin Hsiang",
+        "name": "Unsettling Signs",
+        "pack_code": "tdcc",
+        "position": 56,
+        "quantity": 1,
+        "stage": 1,
+        "text": "[action] Spend 1 or more clues, as a group: For each clue spent, discard the top 5 cards (10 cards instead if there is 1 investigator) of the encounter deck and draw each location discarded by this effect.\n<b>Objective</b> - If each surviving investigator is at a [[Sanctum]] location, investigators may spend the requisite clues, as a group, to advance.<i>(Hint: you may not be able to explore further after you advance.)</i>",
+        "type_code": "act"
+    },
+    {
+        "back_name": "Fate of the Pilgrims",
+        "back_text": "<b>If there are 3 or more [[Cultist]] cards under this act::</b>\n<blockquote>The tunnels buckle and spasm as you flee the apiary. You usher the last of the surviving pilgrims out as the alien parasites screech in primal rage behind you You are safe... for now.</blockquote>\n<b>(→R1)</b> <i>(page 23)</i>.<hr><b>Otherwise:</b>\n<blockquote>The tunnels buckle and spasm as you flee the apiary. You hear the pilgrims' agonized screams behind you. Suddenly the screams stop, replaced by the sound of gnashing mouths and tearing flesh. You were one of the fortunate ones.</blockquote>/n<b>(→R2)</b> <i>(page 23)</i>.",
```

Comment:
```suggestion
        "back_text": "<b>If there are 3 or more [[Cultist]] cards under this act:</b>\n<blockquote><i>The tunnels buckle and spasm as you flee the apiary. You usher the last of the surviving pilgrims out as the alien parasites screech in primal rage behind you. You are safe... for now.</i></blockquote>\n<b>(→R1)</b> <i>(page 23)</i>.<hr><b>Otherwise:</b>\n<blockquote><i>The tunnels buckle and spasm as you flee the apiary. You hear the pilgrims' agonized screams behind you. Suddenly the screams stop, replaced by the sound of gnashing mouths and tearing flesh. You were one of the fortunate ones.</i></blockquote>\n<b>(→R2)</b> <i>(page 23)</i>.",
```